recipes-core/tdx-enc-handler/tdx-enc-handler/tdx-enc-caam.sh

```diff
@@ -61,6 +67,27 @@ tdx_enc_key_gen() {
     fi
 }
 
+# backup original data in partition (if not encrypted)
+tdx_enc_backup_data() {
+    if [ ${TDX_ENC_PRESERVE_DATA} -eq 1 ]; then
+        mkdir -p "${TDX_ENC_STORAGE_MOUNTPOINT}"
+        if mount -t ext4 ${TDX_ENC_STORAGE_LOCATION} "${TDX_ENC_STORAGE_MOUNTPOINT}"; then
+            tdx_enc_log "Backing up original content..."
+
+            TDX_ENC_PRESERVE_LOCATION=$(mktemp --suffix .tar.bz2)
```

Comment:
The busybox version of `mktemp` does not seem to have this `--suffix` option. It's best not to rely on it; there's no need to use a file name extension anyways.

Reply:
Will remove and test on my dev board today.

Reply:
@rborn-tx - Are you happy to resolve this one as the latest code has this change.